README.md

```diff
@@ -36,8 +36,95 @@ when you view the markdown file in GitHub unless you view the raw text.
 <!-- TODO: After you have finished customizing the templates, remove this line and everything above
 from the README.md leaving only your finished README content for your project. -->
 
----
+<!-- Template begins here -->
```

Comment:
I would rather have the README template be a separate file called README-template.md.  I think otherwise it's going to be too confusing with the project-template's actual README.

@carolynvs thoughts?

Reply:
I agree with renaming it to README-template.md. I think that's a bit cleaner, but it does make it inconsistent with the other templates, which just have their final names. I'm ok with the inconsistency, but I wanted to at least mention it :)

Also, since @carolynvs is out until the new year, should we hold on this until then or keep moving forward?

Reply:
A lot of the files require a rename for use ... see the GOVERNANCE-* files for example.  And the LICENSE files, once we straighten that out, will require similar.  Not ideal, but I don't know another way to do it.

Reply:
Given that @carolynvs is out, @karenhchu  or @parispittman, comments?

Reply:
I fixed the formatting issue in the governance templates, and I split the README into 2 files (README.md for the template repo and README-template.md for project README files).

If anyone has any other feedback before we send this to our TOC rep for review, please let me know!